docs/source/reference-lowlevel.rst

```diff
@@ -56,6 +56,35 @@ Global statistics
 .. autoclass:: RunStatistics()
 
 
+.. _trio_contexts:
+
+The current Trio context
```

Comment:
```suggestion
Being "in Trio context"
```

Reply:
I think that wording is somewhat (grammatically??) confusing, though I can't exactly pinpoint why. Trio is perfectly fine as an adjective yet my brain hates "in Trio context"...

Reply:
"Trio context" in your version has Trio as an adjective too, FWIW! I think "current Trio context" is a misleading name because the whole point of the section is that there are different things that "Trio context" can refer to, so it's weird to designate a single one of them (run vs task) as "current". But we could go a different direction, such as titling this "Checking for Trio" or something.